packages/core/contracts/financial-templates/long-short-pair/LongShortPairCreator.sol

```diff
@@ -51,64 +68,46 @@ contract LongShortPairCreator is Testable, Lockable {
     /**
      * @notice Creates a longShortPair contract and associated long and short tokens.
      * @dev The caller must approve this contract to transfer `prepaidProposerReward` amount of collateral.
-     * @param expirationTimestamp unix timestamp of when the contract will expire.
-     * @param collateralPerPair how many units of collateral are required to mint one pair of synthetic tokens.
-     * @param priceIdentifier registered in the DVM for the synthetic.
-     * @param longSynthName Name of the long synthetic tokens to be created.
-     * @param longSynthSymbol Symbol of the long synthetic tokens to be created.
-     * @param shortSynthName Name of the short synthetic tokens to be created.
-     * @param shortSynthSymbol Symbol of the short synthetic tokens to be created.
-     * @param collateralToken ERC20 token used as collateral in the LSP.
-     * @param financialProductLibrary Contract providing settlement payout logic.
-     * @param customAncillaryData Custom ancillary data to be passed along with the price request. If not needed, this
+     * @param params Constructor params used to initialize the LSP. Key-valued object with the following structure:
```

Comment:
nit: in the other struct, you put the comments on the struct itself rather than on the method. In this case, you put the comments on the method? Should we make these consistent?

Reply:
I added them to this method as this is method like this as this is the primary interface people will be calling whereas the constructor for the LSP is more of an internal function as we don't want people deploying LPSs directly ideally.

I'll add the comment there as well for consistency :)